tsdb/cursor.go

```diff
@@ -148,13 +148,14 @@ type cursorHeapItem struct {
 
 // TagSetCursor is virtual cursor that iterates over multiple TagsCursors.
 type TagSetCursor struct {
-	measurement string            // Measurement name
-	tags        map[string]string // Tag key-value pairs
-	cursors     []*TagsCursor     // Underlying tags cursors.
-	currentTags map[string]string // the current tags for the underlying series cursor in play
+	measurement   string            // Measurement name
+	currentFields interface{}       // the current decoded and selected fields for the cursor in play
+	tags          map[string]string // Tag key-value pairs
+	cursors       []*TagsCursor     // Underlying tags cursors.
+	currentTags   map[string]string // the current tags for the underlying series cursor in play
 
-	SelectFields []string // fields to be selected
-	Fields       []string // fields to be selected or filtered
+	SelectFields       []string // fields to be selected
+	SelectFilterFields []string // fields to be selected or filtered
```

Comment:
OK, not sure if the comment would be more correct then if it read "fields to be selected AND filtered".